_layouts/base.html.slim

```diff
@@ -367,6 +367,7 @@
       script src="#{site.base_url}/javascripts/a-b-testing.js"
       script src="#{site.base_url}/javascripts/stackoverflow.js"
       script src="#{site.base_url}/javascripts/product-forums.js"
+
     - unless page.extra_javascripts.nil?
```

Comment:
@LightGuard  I didn't remove it, the red back-ground color is coming from Github's error-highlighting feature in the text editor/viewer 😃